main.js

```diff
@@ -1,2 +1,51 @@
+'use strict';
+// HTMLより入力欄、追加ボタン、追加した内容を表示するためにIDを取得
+const addTask = document.getElementById('addtask');
+const submitButton = document.getElementById('submit');
+const todoLists = document.getElementById('todolists');
 
+// 追加するための空配列を定義
+let todos = [];
 
+// 追加ボタンがクリックされたら実行する処理を実装
+submitButton.addEventListener('click', () => {
+  const taskStatus = '作業中';
+  todos.push({
+    id: todos.length,
+    comment: addTask.value,
+    status: taskStatus,
+  });
```

Comment:
12行目でステータスを変数に格納していますが、変数を定義せずに以下のようにしてしまって良いかなと思います。
```suggestion
  todos.push({
    id: todos.length,
    comment: addTask.value,
    status: '作業中',
  });
```

変数を定義する際の考え方としては、
- その変数を複数回使用する予定があるか
- 変数を定義したことにより、可読性が上がるか

という感じです。
今回は変数を定義しなくても、オブジェクトのstatusというkeyに対して値を入れるので、上記のようにしてしまっても可読性は担保されるかな、という考えです！

Reply:
確かに実装した際は違和感なかったですが、このオブジェクト内でしか用いないため無駄な記述だなと気づきました。
ありがとうございます！